src/index.js

```diff
@@ -0,0 +1,106 @@
+// @flow
+/**
+ * Verdaccio auth memory
+ */
+export default class Memory {
+  // flow types
+  config: Object;
+  verdaccioConfig: Object;
+  users: Object;
+  max_users: number;
+  constructor(
+    config: { users: Object },
+    verdaccioConfig: { max_users: number }
```

Comment:
Here you can create your own Flow object extending `Config`, check this https://github.com/verdaccio/verdaccio-google-cloud/blob/master/src/data-storage.js#L24